qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/car/MainCarSession.kt

```diff
@@ -47,7 +46,6 @@ class MainCarSession : Session() {
 
     init {
         logAndroidAuto("MainCarSession constructor")
-        MapboxCarApp.setup()
```

Comment:
This is the reason for this pull request. It is hard to know that you need to call this and I've been working on deleting it.